content/references/xrpljs2-migration-guide.jp.md

```diff
@@ -0,0 +1,344 @@
+---
+html: xrpljs2-migration-guide.html
+blurb: JavaScriptコードをより新しいクライアントライブラリに移行する方法を学びましょう。
+parent: https://js.xrpl.org/
+---
+# ripple-lib 1.xからxrpl.js 2.xへの移行ガイド 
+
+**ripple-lib** (1.x)ライブラリからJavaScript / TypeScriptコードを移行し、代わりにXRP Ledger用**xrpl.js** (2.x)ライブラリーを使用するためには、以下の手順に従ってください。
+
+**ヒント:** 必要な場合は、依然[legacy 1.x "RippleAPI"用ドキュメンテーション](https://github.com/XRPLF/xrpl.js/blob/1.x/docs/index.md)にアクセスできます。
+
+## 差異の概略
+
+xrpl.js v2.0では、多くのフィールドと機能に"新しい"名前があります。より正確には、xrpl.jsは現在、[HTTP / WebSocket APIs](http-websocket-apis.html)と同じ名前を使用しています。XRP Ledgerで実行可能な"OfferCancel"のような[トランザクションタイプ](transaction-types.html)をライブラリが使用する場所では、"orderCancellation"オブジェクトのようなripple-libに特有の構造はなくなりました。ripple-lib 1.xでこれらの構造をリターンする多くのAPIメソッドはなくなりました。2.0では、WebSocket APIと同じフォーマットでリクエスト、レスポンスを行います。
+
+ripple-lib 1.xからの包括的な`RippleAPI`クラスもなくなりました。xrpl.js 2.xでは、ネットワーク運用のための`Client`クラスがあり、その他全ての運用は厳格にオフラインです。アドレスとキーのための新しい`Wallet`クラス、また、トップレベルの`xrpl`オブジェクトの下にその他のクラスとプロパティがあります。
+## 定型文での比較
+
+**ripple-lib 1.10.0:**
+
+```js
+const ripple = require('ripple-lib');
+
+(async function() {
+  const api = new ripple.RippleAPI({
+    server: 'wss://xrplcluster.com'
+  });
+
+  await api.connect();
+
+  // Your code here
+
+  api.disconnect();
+})();
+```
+
+**xrpl.js 2.0.0:**
+
+```js
+const xrpl = require("xrpl");
+
+(async function() {
+  const client = new xrpl.Client('wss://xrplcluster.com');
+
+  await client.connect();
+
+  // Your code here
+
+  client.disconnect();
+})();
+```
+
+
+## バリデーション結果
+
+デフォルトでは、ripple-lib 1.xにおけるほとんどのメソッドは、[コンセンサスプロセス](consensus.html)によって検証された最終結果をリターンするのみでした。xrpl.jsと同等の多くのメソッドは、WebSocket APIをコールするために[`Client.request()`メソッド](https://js.xrpl.org/classes/Client.html#request)を使用します。WebSocket APIにおいて、XRP Ledgerサーバーのデフォルト設定では、検証済みデータだけはなく未検証のデータを含むことがあります。 
+
+[分散型取引所](decentralized-exchange.html)の状態を調べる時のように、完了見込みの多数のトランザクション結果が保留中であるため、現時点のオープンレジャーを使用したい場合があります。また、完了したトランザクション結果を取り込んだ検証済みのレジャーを使用したい場合もあります。
+
+When making API requests with xrpl.js 2.0 using `Client.request()`, you should explicitly [specify what ledger to use](basic-data-types.html#specifying-ledgers). For example, to look up trust lines using the latest _validated ledger_:
```

Comment:
I think the next paragraph translates this sentence so the English can be removed.